docker-entry.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# migration
+prisma migrate dev
```

Comment:
Migration will run on every container start which is unnecessary right? Unless we change some code. Can you make it run only on first start?